projects/ngx-drag-to-select/src/lib/select-item.directive.ts

```diff
@@ -34,6 +34,10 @@ export class SelectItemDirective implements OnInit, DoCheck {
 
   @Input() dtsSelectItem: any | undefined;
 
+  @HostBinding('class.dts-disabled')
+  @Input()
```

Comment:
It's a nitpick but can you please change the order here
```suggestion
  @Input()
  @HostBinding('class.dts-disabled')
```